src/components/Gantt.vue

```diff
@@ -373,9 +370,77 @@
         },
 
         methods: {
+            buildGroupByData() {
+                const position = this.calculate ? this.calculatedPosition : this.position
+                const titleGroupings = { misc: [] }
+                const startObj = this.categoryGroupings && this.categoryGroupings !== true ? this.categoryGroupings : { misc: [] }
+
+                const processNodes = this.repeats.reduce((grouped, item, i, array, sortKey = item.group_by) => {
+                    if (this.categoryGroupings === true && sortKey) {
+                        grouped[sortKey] = grouped[sortKey] || []
+                    }
+
+                    const computedSortKey = grouped[sortKey] ? sortKey : 'misc'
+
+                    const group = grouped[computedSortKey]
+                    if (group.indexOf(item) === -1) group.push(item)
+
+                    titleGroupings[computedSortKey] = titleGroupings[computedSortKey] || []
+
+                    let index = titleGroupings[computedSortKey].indexOf(item.title.toLowerCase())
+                    if (index === -1) {
+                        index = titleGroupings[computedSortKey].length
+                        titleGroupings[computedSortKey].push(item.title.toLowerCase())
+                    }
+                    item.event_index = this.grouping ? index : i
+
+                    if (this.showRepeats && item.children && item.children.length) {
+                        item.children.map((ch) => {
+                            ch.event_index = index
+                            position(ch)
+                            return ch
+                        })
+                    }
+
+                    if (item.dependencies) {
+                        item.dependencies.map((dep) => {
+                            this.links.push([
```

Comment:
We could treat these links like everything else in `buildGroupByData`  and scope it to a category, which would fix links being associated to the wrong blocks and bleeding into other categories as seen below

<img width="1428" alt="screen shot 2018-05-14 at 10 59 59" src="https://user-images.githubusercontent.com/8988409/39991009-0e6e356a-5766-11e8-9c6f-b3497e1fe893.png">